Contribution.md

```diff
@@ -24,20 +24,30 @@ Please note that this installation only need to be done ONCE.
 
 1. Close this repository
 
-2. Install n8n cli
+1. Install n8n cli
 
 ```bash
 # n8n-nodes-formx/
 make vendor
 ```
 
-3. Install packages
+1. Install packages
 
 ```bash
 npm i
+
+```
+
+1. Setup enviroment variable and config
+
+```bash
+# n8n-nodes-formx/
+cp .env.example .env
```

Comment:
Put this into `make setup`. If rick sees this, he could complain why it isnt in the usual 5 make targets he's expecting in all projects 'D

Reply:
> Oh, pandawork deployment is on another repo...Please update formx-integration after this PR is merged :adore:

Yes, I actually doubt should we include those make vendor / make start target in this repo.

Reply:
Well since this is an oursky project, I think it makes sense to make it oursky first